doc/classes/RichTextLabel.xml

```diff
@@ -69,19 +74,21 @@
 			<return type="String">
 			</return>
 			<description>
-				Returns the raw text, stripping out the formatting information.
+				Returns the raw text.
 			</description>
 		</method>
 		<method name="get_total_character_count" qualifiers="const">
 			<return type="int">
 			</return>
 			<description>
+				Returns the total number of characters.
 			</description>
 		</method>
 		<method name="get_v_scroll">
 			<return type="VScrollBar">
 			</return>
 			<description>
+				Return current vertical scroll value.
```

Comment:
Return -> Returns

I don't think this one is correct. It returns a VScrollBar not a value.

Reply:
How about "Returns a VScrollBar"? Maybe even "Returns a VScrollBar object"?

Reply:
Yes, but that doesn't really tell us anything - what's the VScrollBar object for? I haven't used that function, so I'm not sure, but I suspect it's the RichTextLabel's vertical scrollbar object, so that you can access its members, properties, etc. Probably worth testing it out.

Reply:
For now, I will leave that out until it is tested.